test/integration/integration_test.py

```diff
@@ -0,0 +1,117 @@
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from re import compile, Pattern
+from subprocess import check_call, check_output
+
+base_command_runtime: str = "velocitas exec runtime-k3d"
+base_command_deployment: str = "velocitas exec deployment-k3d"
+
+kubectl_regs = {
+    compile(r".*Kubernetes control plane is running"): False,
+    compile(r".*CoreDNS is running"): False,
+    compile(r".*Metrics-server is running"): False
+}
+
+dapr_regs = {
+    compile(r".*dapr-placement-server\s+dapr-system\s+True\s+Running.*"): False,
+    compile(r".*dapr-dashboard\s+dapr-system\s+True\s+Running.*"): False,
+    compile(r".*dapr-sentry\s+dapr-system\s+True\s+Running.*"): False,
+    compile(r".*dapr-sidecar-injector\s+dapr-system\s+True\s+Running.*"): False,
+    compile(r".*dapr-operator\s+dapr-system\s+True\s+Running.*"): False
+}
+
+image_reg = {compile(r"localhost:12345/sampleapp\s+local.+"): False}
+
+pods_regs = {
+    compile(r".*mqtt-broker-.+-.+\s+1/1\s+Running.*"): False,
+    compile(r".*bash-.+-.+\s+1/1\s+Running.*"): False,
+    compile(r".*seatservice-.+-.+\s+2/2\s+Running.*"): False,
+    compile(r".*feedercan-.+-.+\s+2/2\s+Running.*"): False,
+    compile(r".*vehicledatabroker-.+-.+\s+2/2\s+Running.*"): False,
+    compile(r".*zipkin-.+-.+\s+1/1\s+Running.*"): False,
+    compile(r".*sampleapp-.+-.+\s+2/2\s+Running.*"): False,
+}
+
+uninstall_runtime_reg = {
+    compile(r"release \"vehicleappruntime\" uninstalled"): False
+}
+
+reset_controlplane_reg = {
+    compile(r"Successfully deleted cluster cluster!"): False
+}
+
+
+def escape_ansi(line):
+    ansi_escape = compile(r'(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]')
+    return ansi_escape.sub('', line)
+
+
+def check_regs(regs: dict, output: str) -> bool:
+    final_flag = True
+    # optional print
+    print(output)
+    for reg in regs:
+        if not regs[reg]:
+            if reg.search(output):
+                regs[reg] = True
+        final_flag = final_flag and regs[reg]
+    return final_flag
+
+
+def check_cluster_status_and_dapr(kubectl_regs: dict, dapr_regs: dict) -> bool:
+    command_logs_kubectl = check_output(["kubectl", "cluster-info"]).decode()
+    command_logs_dapr = check_output(["dapr", "status", "-k"]).decode() 
+    command_logs_kubectl = escape_ansi(command_logs_kubectl)
+    return check_regs(kubectl_regs, command_logs_kubectl) and check_regs(dapr_regs, command_logs_dapr)
+
+
+def deploy_runtime_check(kubectl_regs: dict, dapr_regs: dict) -> bool:
```

Comment:
function naming does not match the other `check_*` functions. Please align this.